src/state/markets/test_market.rs

```diff
@@ -1402,6 +1402,60 @@ fn test_fok_and_ioc_limit_5() {
         )
         .is_none());
 
+    let mut mock_clock_fn = || (1000, 1000);
+
+    assert_eq!(
+        market
+            .place_order(
+                &taker,
+                OrderPacket::new_ioc(
+                    Side::Bid,
+                    None,
+                    100,
+                    0,
+                    0,
+                    0,
+                    SelfTradeBehavior::Abort,
+                    None,
+                    rng.gen::<u128>(),
+                    false,
+                    Some(2),
+                    None
+                ),
+                &mut record_event_fn,
+                &mut mock_clock_fn,
+            )
+            .unwrap()
+            .1,
+        MatchingEngineResponse::default()
+    );
+
+    assert_eq!(
+        market
+            .place_order(
+                &taker,
+                OrderPacket::new_ioc(
+                    Side::Bid,
+                    None,
+                    100,
+                    0,
+                    0,
+                    0,
+                    SelfTradeBehavior::Abort,
+                    None,
+                    rng.gen::<u128>(),
+                    false,
+                    None,
+                    Some(2),
+                ),
+                &mut record_event_fn,
+                &mut mock_clock_fn,
+            )
+            .unwrap()
+            .1,
+        MatchingEngineResponse::default()
+    );
```

Comment:
Add one more test that includes a successful IOC with TIF params set in the future

Reply:
i added this and verified tests still pass